src/couch_views/src/couch_views_indexer.erl

```diff
@@ -236,20 +267,26 @@ process_changes(Change, Acc) ->
 
     IncludeDesign = lists:keymember(<<"include_design">>, 1, DesignOpts),
 
-    Acc1 = case {Id, IncludeDesign} of
-        {<<?DESIGN_DOC_PREFIX, _/binary>>, false} ->
-            maps:merge(Acc, #{
-                count => Count + 1,
-                last_seq => LastSeq
-            });
-        _ ->
-            Acc#{
-                doc_acc := DocAcc ++ [Change],
-                count := Count + 1,
-                last_seq := LastSeq
-            }
-    end,
-    {ok, Acc1}.
+    DocVS = fabric2_fdb:next_vs(fabric2_fdb:seq_to_vs(LastSeq)),
+    case BuildToVS == true andalso ViewVs =< DocVS of
```

Comment:
Correct, build_to_vs is just the flag to know if the indexer must only build to a specific vs. We get the actual vs via `the couch_views_fdb:get_build_vs`

> Will have to look, but overloading view_seq also likely means that restarts probably don't work? How would we track where the build is at if the job has to restart?

Oop that is a bug. I've changed it so that you can set `LastSeq = false` for the write_docs and it won't update the build sequence. That way when the mango_indexer updates a doc while its still being built it won't clash.